swarm.go

```diff
@@ -374,11 +374,24 @@ func (s *Swarm) Start(srv *p2p.Server) error {
 	log.Info("Updated bzz local addr", "oaddr", fmt.Sprintf("%x", newaddr.OAddr), "uaddr", fmt.Sprintf("%s", newaddr.UAddr))
 
 	if s.config.SwapEnabled {
-		err := s.swap.Deploy(context.Background(), s.backend, s.config.Path)
-		if err != nil {
-			return fmt.Errorf("Unable to deploy swap contract: %v", err)
+		if s.config.Contract != (common.Address{}) {
```

Comment:
no need for additional brackets here

Reply:
Actually, this seems to be needed.
If I don't have brackets it complains at the opening bracket:
`syntax error: unexpected { at end of statement` and the else branch:
`syntax error: unexpected else after top level declaration`
I adopted this style from line `swarm.go:302`
```golang
if addr != (common.Address{}) {
```
which indeed resolves the error.